### PR TITLE
SW-4932 Add CohortListView and CohortView with basic table and navigation

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#2218** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2218?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2218?utm_source=stack-comment).

File: src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx [hunk 1]
```diff
@@ -47,10 +47,10 @@ export default function AcceleratorCohortNewView(): JSX.Element {
       // set snackbar with status
       snackbar.toastSuccess(strings.formatString(strings.COHORT_ADDED, cohort.name) as string);
 
-      // navigate to accelerator overview
-      goToAcceleratorOverview();
+      // navigate to cohorts list
+      goToCohortsList();
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx:51]: Should we be going to the newly created cohort view? Just to to stick with current pattern.

File: src/scenes/AcceleratorRouter/NavBar.tsx [hunk 2]
```diff
@@ -67,6 +68,14 @@ export default function NavBar({ backgroundTransparent, setShowNavBar }: NavBarP
 
       <NavSection />
 
+      <NavItem
+        icon='organizationNav'
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/NavBar.tsx:72]: capture icon requirement here https://terraformation.atlassian.net/browse/SW-4885

File: src/scenes/AcceleratorRouter/CohortView.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      {
+        name: activeLocale ? strings.COHORTS : '',
+        to: APP_PATHS.ACCELERATOR_COHORTS,
+      },
+    ],
+    [activeLocale]
+  );
+
+  return (
+    <Page crumbs={crumbs} title={cohort?.name || ''} rightComponent={rightComponent}>
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/CohortView.tsx:56]: Oh wow, didn't realize we had a Page component.. I created another one recently 🤦

**Review by karthikbtf — APPROVED** [review]
looks good, one comment about redirection

File: src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx (hunk 1 above)
Comment (nickgraz) [reply src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx:51]: Ah, yes, good catch

File: src/scenes/AcceleratorRouter/CohortView.tsx (hunk 3 above)
Comment (nickgraz) [reply src/scenes/AcceleratorRouter/CohortView.tsx:56]: Hehe yea I feel like there are a few floating around...

File: src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx (hunk 1 above)
Comment (nickgraz) [reply src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx:51]: Gonna make this change in #2219 to avoid conflicts

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 27, 7:18 PM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2218).

**Pull request merged**